tests/SshTest.php

```
@@ -164,3 +164,10 @@
 
     assertMatchesSnapshot($command);
 });
+
+it('can login with a password', function () {
```

could you also provide a fail scenario with an invalid password?

---

sorry i'm not at home right now i couldn't test it right now